pyanaconda/modules/users/installation.py

```diff
@@ -161,3 +163,33 @@ def run(self):
     def _set_ssh_keys(self):
         for key_data in self._ssh_key_data_list:
             users.set_user_ssh_key(key_data.username, key_data.key)
+
+class ConfigureRootPasswordSSHLogin(Task):
+    """Optionally add an override allowing root to login with password over SSH."""
+
+    SSHD_OVERRIDE_DROP_DIR = "etc/systemd/system/sshd.service.d/"
+    OVERRIDE_CONFIG_NAME = "anaconda.conf"
+
+    def __init__(self, sysroot, root_password_ssh_login_allowed):
```

Comment:
Please, make the name of the parameter shorter. The meaning is clear from the description of the class.

Reply:
Indeed, that can be shorter.